maven-plugin/src/main/java/org/openmrs/maven/plugins/packager/config/ValidateContentPackageMojo.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.maven.plugins.packager.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import com.vdurmont.semver4j.Semver;
+import com.vdurmont.semver4j.SemverException;
```

Comment:
I pointed this out elsewhere, but we use [this SemVer](https://github.com/zafarkhaja/jsemver) library in the SDK. I think we should stick to that one.

Reply:
Basically, we should have one definition of "valid SemVer"

Reply:
> I pointed this out elsewhere, but we use [this SemVer](https://github.com/zafarkhaja/jsemver) library in the SDK. I think we should stick to that one.

Changed. Yes, you did mention about this library, thought you wanted me to use something like that and not use that.

semver4j was mentioned in https://www.baeldung.com/java-comparing-versions